client/tests/localmode.py

```diff
@@ -108,3 +110,159 @@ def get_training_set_from_file(file, entity, feature_col, label, name_variant):
     label = get_label(df, entity, label)
     training_set_df = get_training_set(label, feature, entity)
     return training_set_df.values.tolist()
+
+class TestLocalMode:
+    def test_duplicate_resources(self):
+        ff.register_user("featureformer").make_default_owner()
+
+        local = ff.register_local()
```

Comment:
Is there a reason for the duplicated code?

Reply:
Yes we want to make sure that the code doesn't break/error when the same resource is registered again. Thats why we need duplicated code